server/plugin.go

```diff
@@ -308,8 +308,12 @@ func (p *Plugin) OnActivate() error {
 			}
 
 			status, apiErr := p.API.GetPluginStatus(autolinkPluginId)
-			if apiErr != nil || status.State != model.PluginStateRunning {
-				p.API.LogWarn("OnActivate: Autolink plugin is unavailable", "error", err.Error(), "status", status)
+			if apiErr != nil {
+				p.API.LogWarn("OnActivate: Autolink plugin unavailable. API returned error", "error", apiErr.Error())
+				continue
+			}
+			if status.State != model.PluginStateRunning {
+				p.API.LogWarn("OnActivate: Autolink plugin unavailable. Plug-in is not running", "status", status)
```

Comment:
We hardly ever refer to `plugin` as `plug-in`, so I suggest removing the `-` in this message

```suggestion
				p.API.LogWarn("OnActivate: Autolink plugin unavailable. Plugin is not running", "status", status)
```

Reply:
Sounds sensible. Just changed.